### DISCUSSION TITLE
Page after payment/donation not showing properly

### DISCUSSION THREAD

**Discussion by PatAbah** [Acode-Foundation/Acode#665] [General]
First of all, this update was a bomb. Great work!

So, I donated some days ago and the "thank you" page/modal didn't display properly. See the screenshot I've attached. 
![IMG_20230313_153631](https://user-images.githubusercontent.com/34959120/225343781-ef98ce7e-ea25-45b6-a63d-c964b92e074f.jpg)

P.S: I noticed that "Dracula Pro" was no longer in the list of editor themes. It was kind of my favourite. 

Lastly, when reorganising tab, as in after long pressing it and trying to move the tab, the movement misbehaves. I started noticing the weird behaviour from v1.6...i think. Please note that this misbehavior only happens when I leave my finger on the tabs area while scrolling.

**Comment by deadlyjack** [comment]
Thanks for reporting it, i'll fix it soon.
